demo/Makefile

```diff
@@ -48,13 +41,14 @@ build-snapshot:	build-proxy
 
 DOCKER_RUN=docker run --rm -p 8080:8080 -v /var/run/docker.sock:/var/run/docker.sock
 
-run:	build
-	$(DOCKER_RUN) $(IMAGE):$(TAG)
+run:
+	docker-compose up
 
 run-snapshot:	build-snapshot
 	$(DOCKER_RUN) $(IMAGE):SNAPSHOT
 
 clean:
+	docker-compose down
```

Comment:
Better than nothing, though still a bit of a regression compared to the base state where Ctrl-C cleans up.